### ISSUE TITLE
Notify when password isn't complex enough

### ISSUE THREAD

**Issue by heyarne** [nextcloud/server#7619]
I was just trying to create a user and it didn't work. I did not receive any kind of notification, it just didn't work. Opening the developer tools gave me a 403 status code with a rendered html response telling me access forbidden, so I logged out and in again, assuming my session expired without me notificng it. It turned out that what solved the problem was [this forum post](https://help.nextcloud.com/t/cant-add-new-users/6800).

Is this expected behavior? If yes, can it maybe be changed to show a notice and password requirements?

**Comment by dirigit** [comment]
Hi,

some kind of message or information would be nice. I had same problem and got it solved using another password.

You can change requirenments in settings (security, scroll down a bit).

cu, diri

**Comment by tflidd** [comment]
Are you using nginx?
https://github.com/nextcloud/password_policy/issues/32#issuecomment-293596764

**Comment by heyarne** [comment]
I *am* using nginx, however I'm now receiving a 503 error in a similar way. The (invisible) response again is a rendered HTML page containing mainly this:

```
       <ul class="error-wide">
		<li class='error'>Bad Signature<br></li>
	</ul>
```

I ended up changing the password via `sudo -u www-data ./occ user:resetpassword $user`.

**Comment by dirigit** [comment]
Hi,

I'm using Apache.

**Comment by MorrisJobke** [comment]
@heyarne Could you check your nginx config against https://docs.nextcloud.com/server/12/admin_manual/installation/nginx.html ? Because we have seen similar problems in the past.

Usually it should show a yellow banner on the top of the page if something is not working.

@dirigit Could you also double check your Apache config. Could be a similar problem as in the linked comment by @tflidd

**Comment by heyarne** [comment]
There were indeed some differences in my config and the one on the site you linked to. I changed my config to look like [this](https://gist.github.com/heyarne/90cfd65d1de2cb617b29c6d5cb245bd6). Still, when I try to change the password for user *user* to *user* (something too simple), the only feedback I get is in the Firefox Developer tools. After authorizing as an admin that I really want to change user's password I receive a `503` response with the same rendered HTML template containing the `bad signature` error message that I described in my second comment on this issue.
